library/src/main/java/com/google/android/exoplayer2/audio/AudioTrack.java

```diff
@@ -443,6 +443,14 @@ public void configure(String mimeType, int channelCount, int sampleRate,
     }
 
     boolean passthrough = !MimeTypes.AUDIO_RAW.equals(mimeType);
+    if (Util.SDK_INT <= 23 && "foster".equals(Util.DEVICE) && "NVIDIA".equals(Util.MANUFACTURER)) {
```

Comment:
nit - Seems like the new code belongs at L444 as part of the block that's setting `channelConfig`. It's currently sandwiched between the line where `passthrough` is calculated and the line where it's used.